RiotSwiftUI/Modules/Template/SimpleUserProfileExample/View/TemplateUserProfileHeader.swift

```diff
@@ -0,0 +1,58 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+@available(iOS 14.0, *)
+struct TemplateUserProfileHeader: View {
+    
+    // MARK: - Properties
+    
+    // MARK: Public
+    
+    @Environment(\.theme) var theme: ThemeSwiftUI
+    let avatar: AvatarInputProtocol?
+    let displayName: String?
+    let presence: TemplateUserProfilePresence
+    
+    var body: some View {
+        VStack {
+            if let avatar = avatar {
+                HStack{
+                    Spacer()
+                    AvatarImage(avatarData: avatar, size: .xxLarge)
+                    Spacer()
+                }
+                .padding(.vertical)
```

Comment:
Same as above this could become
```suggestion
                AvatarImage(avatarData: avatar, size: .xxLarge)
                    .padding(.vertical)
```